Initialize data catalog overview docs with data flow diagram (#102)
ccao-data/data-architecture: dbt/models/overview.md

{% docs __overview__ %}

> # Cook County Assessor Data Department Catalog

Author: The text here is mostly placeholder text so far, but I'm open to edits nonetheless!

> This site documents the data that are built and maintained by the Data
> Department of the Cook County Assessor's Office.
> 
> These docs are under active development and are generated automatically using
> [dbt](https://docs.getdbt.com/docs/introduction). You can view the source code
> on [GitHub](https://github.com/ccao-data/data-architecture/).

Reviewer: ```suggestion
This site documents the data infrastructure created and used by
the Data Department of the Cook County Assessor's Office.

These docs are under active development and are generated automatically using
[dbt](https://docs.getdbt.com/docs/introduction). You can view the source code
on [GitHub](https://github.com/ccao-data/data-architecture/).
```

## Data Flow Diagram

> The following diagram provides a summary of the way in which the Data
> Department builds and uses our data warehouse.

Reviewer: ```suggestion
The following diagram summarizes the Data Department's current infrastructure and data flows.
```
